oada-core/http-handler/oada-srvc-http-handler/websockets.ts

```diff
@@ -0,0 +1,485 @@
+import { resolve } from 'path'
+import { strict as _assert } from 'assert'
+
+import debug from 'debug'
+
+import * as jsonpointer from 'jsonpointer'
+import Ajv from 'ajv'
+
+import { OADAError } from 'oada-error'
+
+import * as WebSocket from 'ws'
+// prettier-ignore
```

Comment:
This should be fixed in `prettier` 2.0. Another reason not use the standard wrapper.